docker-targets.mk

```diff
@@ -18,5 +18,7 @@ define docker-push-to-registry
 endef
 endif
 
+docker-build: GOOS=linux
+docker-build: GOARCH=amd64
 docker-build: build ## Build container image
-	docker build ${DOCKER_BUILD_EXTRA_PARAMS} -t ${IMG} -t ${IMG}:${VERSION} -f Dockerfile $(_DOCKER_TARGETS_MK_DIR)
```

Comment:
because binaries are now built outside the containers, we don't have to add all workspace modules anymore and can therefore build locally again